tests/test_orders.py

```diff
@@ -52,4 +52,17 @@ def test_get_order_by_uuid(self):
         assert(actual_order["order_number"] == unique_order_number)
         assert(actual_order["uuid"] == created_order.uuid)
 
+    def test_update_order(self):
+        unique_order_number = random.randint(1,1000000)
+        created_order = OrderActions.create( "OH",  unique_order_number , "OH06")
+        actual_order=OrderActions.get_order_by_uuid(created_order.uuid)
```

Comment:
Call this again to make sure it runs the SQL query from the ORM/database again and is not just referencing a cached value.